src/cart/crt_hg.c

```diff
@@ -1364,8 +1364,8 @@ crt_hg_req_send_cb(const struct hg_cb_info *hg_cbinfo)
 		crt_cbinfo.cci_rc = rc;
 
 		if (crt_cbinfo.cci_rc != 0)
-			RPC_CERROR(crt_quiet_error(crt_cbinfo.cci_rc), DB_NET, rpc_priv,
-				   "RPC failed; rc: " DF_RC "\n", DP_RC(crt_cbinfo.cci_rc));
+			RPC_WARN(rpc_priv,
+				 "RPC failed; rc: " DF_RC "\n", DP_RC(crt_cbinfo.cci_rc));
```

Comment:
we need something like RPC_CWARN here that takes the same condition. you dont want to print warnings for all group version mismatches either, and those are very frequent

Reply:
For sure.